schema/fields/event_status.js

```diff
@@ -0,0 +1,13 @@
+import { GraphQLEnumType } from 'graphql';
+
+export default {
+  type: new GraphQLEnumType({
+    name: 'EventStatus',
+    values: {
+      'current': { value: 'current' },
+      'running': { value: 'running' },
+      'closed': { value: 'closed' },
+      'upcoming': { value: 'upcoming' }
+    }
+  })
+}
```

Comment:
Not really a field. We really need a better file organization system. Should look around for some non-trivial GraphQL schemas in the wild.